api/v1alpha1/ingressnodefirewallnodestate_types.go

```diff
@@ -24,21 +24,33 @@ import (
 
 // IngressNodeFirewallNodeStateSpec defines the desired state of IngressNodeFirewallNodeState.
 type IngressNodeFirewallNodeStateSpec struct {
-	// ingress is a list of ingress firewall policy rules.
+	// interfaceIngressRules is a map that matches interface names to ingress firewall policy rules that shall be
+	// applied on the given interface.
+	// An empty map indicates no ingress firewall rules shall be applied, i.e allow all incoming traffic.
```

Comment:
will this ever happen empty list we have minItems to interfaces and ingress to 1